docs/core/tutorials/cli-templates-create-item-template.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Create an item template for dotnet new
+description: Learn how to create an item template for the dotnet new command. Item templates can contain any number of files.
+author: thraka
```

Comment:
add ms.author as well, otherwise it's me by default 😄

Reply:
hahaha ok.